dvc/external_repo.py

```diff
@@ -62,16 +63,29 @@ def _external_repo(url=None, rev=None, cache_dir=None):
 
     # Adjust new clone/copy to fit rev and cache_dir
     repo = Repo(new_path)
+    # Adjust original repo for pointing remote towards its' cache
+    original_repo = Repo(url)
```

Comment:
This won't work if url is not a local existing directory. It should really be under `if os.path.isdir(url)..` down below.